### ISSUE TITLE
[u/user/permission] Adding the root user to a group does not work.

### ISSUE THREAD

**Issue by zimsneexh** [VelocityVMM/Velocity#8]
Example request:
```
{
  "authkey":"...",
  "uid":0,
  "gid":1,
  "permission":"velocity.user.create"
}
```
Returns "200 OK", but no membership is being created. This only seems to affect the root user, other users work fine.

**Comment by zimsneexh** [comment]
Figured it out (Pretty sure) - Since the user already has all permissions from the parent group,only the "root" membership will be listed in the u/user Endpoint. Since it would be beneficial to display all permissions, I think this should be changed. The "membership" table seemingly already contains the dataset I would expect.

**Comment by maxkofler** [comment]
This might be related to a possible bug in checking if the permission does already exist. It is well possible that it uses the recursive instead of the direct permission check. I will look into this issue...

**Comment by maxkofler** [comment]
This issue is resolved with 51873a0. I already cherry-picked it into main so it is immediately available.

**Comment by maxkofler** [comment]
Aftermath: As I already guessed, this had to do with the recursive checking for permissions. This bug would surface on other users than `root`, too.
